packages/teams-js/src/public/app/prefetch.ts

```diff
@@ -0,0 +1,3 @@
+import { prefetchOriginsFromCDN } from '../../internal/validOrigins';
+
+prefetchOriginsFromCDN();
```

Comment:
Could this file be in the internal folder instead of private/public? Doesn't seem like we intended for anyone to call it and are just depending on it calling a function in the global namespace

Reply:
Yup I could move this there, I would just need to import it somewhere for it to be picked up as a module.